packages/flutter/test/cupertino/action_sheet_test.dart

```diff
@@ -1956,6 +1956,41 @@ void main() {
     );
   });
 
+  testWidgets('Hovering over Cupertino action sheet action updates to the passed cursor on desktop', (WidgetTester tester) async {
+    const SystemMouseCursor customCursor = SystemMouseCursors.click;
+
+    await tester.pumpWidget(
+        createAppWithButtonThatLaunchesActionSheet(
+            CupertinoActionSheet(
+              title: const Text('The Title'),
+              message: const Text('Message'),
+              actions: <Widget>[
+                CupertinoActionSheetAction(
+                  onPressed: () { },
+                  mouseCursor: customCursor,
+                  child: const Text('One'),
+                ),
+              ],
+            )
+        )
+    );
+
+    await tester.tap(find.text('Go'));
+    await tester.pump();
+
+    final TestGesture gesture = await tester.createGesture(kind: PointerDeviceKind.mouse, pointer: 1);
+    await gesture.addPointer(location: const Offset(10, 10));
+    await tester.pumpAndSettle();
+
+    final Offset actionSheetAction = tester.getCenter(find.text('One'));
+    await gesture.moveTo(actionSheetAction);
+    await tester.pumpAndSettle();
+    expect(
+      RendererBinding.instance.mouseTracker.debugDeviceActiveCursor(1),
+      kIsWeb ? SystemMouseCursors.click : customCursor,
```

Comment:
If the test is checking that the mouse cursor provided is always respected, then it shouldn't differ in behavior on web vs on other platforms.

Reply:
will change it so ensure the mouseCursor is always respected instead of checking platform